src/documentation/ToolkitDocumentation/ToolkitDocumentation.tsx

```diff
@@ -60,28 +60,45 @@ const parseAnchor = (anchor: Anchor | undefined): ToolkitNavigationState => {
 };
 
 interface ActiveToolkitLevelProps extends ToolkitProps {
+  anchor: Anchor | undefined;
   state: ToolkitNavigationState;
   onNavigate: (state: ToolkitNavigationState) => void;
   direction: "forward" | "back" | "none";
 }
 
 const ActiveToolkitLevel = ({
+  anchor,
   state,
   onNavigate,
   toolkit,
   direction,
 }: ActiveToolkitLevelProps) => {
-  const topic = state.topicId
-    ? toolkit.contents?.find((t) => t.name === state.topicId)
-    : undefined;
+  const topic = toolkit.contents?.find((t) => {
+    if (!state.topicId) {
+      return undefined;
+    }
+    if (t.slug.current === state.topicId) {
+      return t;
+    }
+    const parentTopic = t.contents?.find(
+      (e) => e.slug.current === state.topicId
+    )?.parent;
+    return parentTopic || undefined;
+  });
+
   const activeItem =
     topic && state.itemId
-      ? topic.contents?.find((i) => i.name === state.itemId)
+      ? topic.contents?.find((i) => i.slug.current === state.itemId)
+      : topic && !state.itemId
+      ? topic.contents?.find((i) => i.slug.current === state.topicId)
       : undefined;
```

Comment:
There may be a better approach to this. Currently handles routes in the form of `?tab=explore&explore=scope-changing-global-variables-in-functions` and `?tab=explore&explore=functions/scope-changing-global-variables-in-functions` which may be useful for external links pointing at the editor.

Reply:
Refactored in commit c02878d